backend/tests/cellMove/selectCell.test.js

```diff
@@ -35,6 +35,7 @@ describe('Select a cell', () => {
       },
```

Comment:
I restructured the test a bit because we've got a lot going on in this page and the tests started to look messy.